chart/identity-api/values.yaml

```diff
@@ -31,6 +31,25 @@ config:
     # - "1.2.3.4/32"
     # - "1.2.3.0/24"
 
+  events:
+    nats:
+      # url is the event server connection url
+      url: ""
+      # source defines the source of the events (defaults to application name)
+      source: ""
+      # publishPrefix is the prefix to add to all published events
+      publishPrefix: ""
+      # connectTimeout is event connection timeout
+      connectTimeout: "10s"
+      # shutdownTimeout is the shutdown grace period
+      shutdownTimeout: "5s"
+      # tokenSecretName is the secret to load the auth token
+      tokenSecretName: ""
```

Comment:
looks like this was not mounted in the container

Reply:
[added](https://github.com/infratographer/identity-api/pull/325/files#diff-ed12cee3b3a8e406cd7f7199eb0c50d2cb57718c9ff48461ace9e3b6a5a6fa9bR97-R104)